README.md

```diff
@@ -1,3 +1,7 @@
 # prune-git-branches
 This script will prune your local branches no longer tracked on remote. <br>
 Run with ```-f``` flag to also remove unmerged branches.
+
+### remove-feature-branches.sh
```

Comment:
Please rename script accordingly so its purpose is clear e.g. remove-non-primary-branches.sh